draft-ietf-ice-rfc5245bis.xml

```diff
@@ -541,8 +541,8 @@ candidate pairs to be used for media.
 When nominating, the controlling agent lets the checks continue
 until at least one valid candidate pair for each media stream is
 found and then chooses a candidate pair from the valid candidate pairs
-and sends a STUN request on the selected pair with a flag set to indicate
-to the controlled peer that it has nominated the selected pair.
+and sends a STUN request on the pair with a flag*** set to indicate
```

Comment:
What do the the `***`s indicate?

Reply:
Sorry, will be removed.  I used it as a "bookmark" when editing :)